docs/tikv-storage-configuration.md

```diff
@@ -0,0 +1,108 @@
+---
+title: TiKV Storage Configuration 
+summary: Learn how to configure TiKV Storage.
+category: operations
+---
+
+# TiKV Storage Configuration
+
+In TiKV, Storage is the component responsible for handling read and write requests. Note that if you are using TiKV with TiDB, most read requests are handled by the Coprocessor component instead of Storage.
+
+## Configuration
+
+There are two sections related to Storage: `[readpool.storage]` and `[storage]`.
+
+### `[readpool.storage]`
+
+This configuration section mainly affects storage read operations. Most read requests from TiDB are not controlled by this configuration section. For configuring the read requests from TiDB, see [Coprocessor configurations](coprocessor-config.md).
```

Comment:
The link is broken? Maybe it should be `tikv-coprocessor-config.md`. BTW this markdown can be named as `tikv-storage-config.md` instead of `tikv-storage-configuration.md` to be unified.